tmk_core/protocol/nrf.mk

```diff
@@ -21,8 +21,10 @@ endif
 
 ifeq ($(MCU_SERIES), NRF52840)
  SRC += $(NRF_DIR)/$(NRF_VER_DIR)/usbd.c \
-       $(NRF_DIR)/$(NRF_VER_DIR)/app_usbd_hid_kbd.c \
-       $(NRF_DIR)/$(NRF_VER_DIR)/app_usbd_hid_mouse.c \
+       $(NRF_DIR)/$(NRF_VER_DIR)/usbd_hid_kbd.c \
```

Comment:
When building the slave side of a split keyboard, I get compile errors such as:
```
tmk_core/protocol/nrf/sdk15/usbd.c:755:33: error: expected expression before ')' token
  755 |           if (!get_usb_enabled()) {
      |                                 ^
```

I think because `tmk_core/protocol/nrf/app_ble_func.h` has in this case:
```
#define get_usb_enabled()
```
Rather than a function declaration.